spec/factories/organisations_factory.rb

```diff
@@ -12,9 +12,12 @@
     # One trait per type named in format : as_xxx e.g   as_local_authority, as_limited_liability_partnership etc
     FloodRiskEngine::Organisation.org_types.each_key do |ot|
       trait :"as_#{ot}" do
-        org_type ot.to_s
-        name ""
-        registration_number ""
+        org_type            { ot.to_s }
+        name                { "" }
+        # this errors with `unknown attribute`
```

Comment:
If this is no longer required, I'd rather we remove it entirely than just leave it commented out.

Reply:
Removed! :-)